spec/inspect_spec.lua

```diff
@@ -411,4 +411,15 @@ describe( 'inspect', function()
       end)
     end)
   end)
+
+  it('allows changing the global tostring', function()
+    local save = _G.tostring
+    _G.tostring = function(x) 
```

Comment:
This can be simplified to `_G.tostring = inspect`. I am not sure however that using `_G` this way is correct idiom for Lua 5.3. Can you check it out?